### DISCUSSION TITLE
Opening "Search project files" and typing fast leaves some characters in the editor

### DISCUSSION THREAD

**Discussion by danielfrg** [zed-industries/zed#8808] [Feature Requests]
Sometimes when I do cmd+P the dialog takes a moment to appear.
If i know the name of the file i am trying to open i am used to start typing immediately. In some cases the cursor is still on the editor and the first character(s) stay on the editor and not as part of searching the file which is double terrible, the file now has typos and the search is incomplete.

Is there any config I can change to improve this? Is becomes really annoying. It's not a huge project and stuff like `node_modules` is being ignored.

pd: awesome project! keep up the great work!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
